tests/protocols/test_websocket.py

```diff
@@ -584,7 +584,7 @@ async def connect(url):
     async with run_server(config):
         with pytest.raises(websockets.exceptions.ConnectionClosed) as exc_info:
             await connect(f"ws://127.0.0.1:{unused_tcp_port}")
-        assert exc_info.value.code == 1006
+        assert exc_info.value.code == 1000 or 1006
```

Comment:
This is silly, will look for  a better way :)

Reply:
This is the definition of their `ConnectionClosedError`:

```A close frame with a code other than 1000 (OK) or 1001 (going away) was received or sent, or the closing handshake didn't complete properly.```

Reply:
I'll leave this up for review but I think it makes sense to keep this way.
I tried changing the test to something like:

```py
async def app(scope, receive, send):
        await send({"type": "websocket.accept"})
        await send({"type": "websocket.send", "code": 1002})
        return 123
```
To cause websockets fail with 1006 again, but that will lead to these lines not being covered:

https://github.com/encode/uvicorn/blob/b54e24874d3765c21f193e2da4c2ac5e6d3f51fe/uvicorn/protocols/websockets/websockets_impl.py#L271-L275